RecoTauTag/Configuration/python/updateHPSPFTaus_cff.py

```diff
@@ -11,30 +11,5 @@
 
 '''
 
-
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationByMVAIsolation2_cff                  import *
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationByIsolation_cfi                      import *
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationByLeadingTrackFinding_cfi            import *
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationAgainstElectron_cfi                  import *
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationAgainstElectronMVA5_cfi              import *
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationAgainstElectronDeadECAL_cfi          import *
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationAgainstMuon_cfi                      import *
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationAgainstMuon2_cfi                     import *
-from RecoTauTag.RecoTau.PFRecoTauDiscriminationAgainstMuonMVA_cfi                   import *
-
-from RecoTauTag.Configuration.HPSPFTaus_cff import hpsPFTauChargedIsoPtSum 
```

Comment:
I haven't tried to check if every single line is needed,
but if you revert back all specified imports (from hpsPFTauChargedIsoPtSum to hpsPFTauDiscriminationByRawPileupWeightedIsolation3Hits) the pat unit tests should work again 
and the miniAOD steps appear to be operational as well.